regions_www/m/pct_shiny_funs.R

```diff
@@ -136,8 +154,10 @@ text_drive_change <- function(purpose){
 }
 
 text_cycle_interzone <- function(purpose){
-  if(purpose=="commute" | purpose=="school")
+  if(purpose=="commute")
     list("cycle" = "Between-zone cyclists*", "*" = "* selected cyclists: see Region Stats tab")
+  else if(purpose=="school")
```

Comment:
Also very neat-looking - will try on actual data but works with just the usual data.